pysoa/test/pytest_fixtures.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import (
+    absolute_import,
+    unicode_literals,
+)
+
+import importlib
+import os
+
+import pytest
+
+from pysoa.client import Client
+from pysoa.test.stub_service import stub_action
+
+
+@pytest.fixture(scope='session')
+def server_settings(server_class):
+    if server_class.use_django:
+        from django.conf import settings
+        settings = settings.SOA_SERVER_SETTINGS
+    else:
+        settings_module = os.environ.get('PYSOA_SETTINGS_MODULE', None)
+        assert settings_module, 'PYSOA_SETTINGS_MODULE environment variable must be set to run tests.'
+        try:
+            thing = importlib.import_module(settings_module)
+            settings = thing.SOA_SERVER_SETTINGS
+        except (ImportError, AttributeError) as e:
+            raise AssertionError('Could not access {}.SOA_SERVER_SETTINGS: {}'.format(settings_module, e))
```

Comment:
So, in the `Server` class, we [permit this non-Django settings module to have _either_ `SOA_SERVER_SETTINGS` _or_ `settings`](https://github.com/eventbrite/pysoa/blob/4f86edea4f535d137ea575d38abdd664381a2774/pysoa/server/server.py#L643-L653). Perhaps we should do the same here.